src/igvfd/schemas/measurement_set.json

```diff
@@ -306,6 +306,22 @@
                 "igvf_utils": "direct RNA, exome capture"
             }
         },
+        "primer_designs": {
+            "title": "Primer Designs",
+            "description": "The primer designs used in this measurement set.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "Primer Design",
+                "type": "string",
+                "linkTo": "TabularFile"
+            },
+            "submissionExample": {
+                "appscript": "[\"john-doe:primer_designs\", \"TSTFF98084731\"]",
+                "igvf_utils": "john-doe:primer_designs, TSTFF98084731"
```

Comment:
I think we should try to use the production style accessions in the submission example.